keps/sig-network/3698-multi-network/README.md

```diff
@@ -0,0 +1,363 @@
+# KEP-3698: Multi-Network
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Personas](#personas)
+    - [Cluster operator](#cluster-operator)
+    - [Application developer](#application-developer)
+  - [Terminology](#terminology)
+  - [User Stories](#user-stories)
```

Comment:
@mskrocki hey o/

Is it too late to raise a new user story ? 

We're mostly interested in having "sticky IPs" for virt workloads; it would basically allow:
- the VM interfaces would get the same IP on migration
- the IP allocation would persist a VM stop. I.e. you stop the VM, but that IP is still reserved for it. Other workloads can not re-use it.

We're checking how to implement this in the kubernetes network plumbing wg, but, if we can prepare the ground to have the feature natively available in Kubernetes, we should take that opportunity.

Reply:
Yeah it is a bit late, we already have draft of Phase I: https://docs.google.com/document/d/17LhyXsEgjNQ0NWtvqvtgJwVqdJWreizsgAZHWflgP-A/edit?usp=sharing

But with the proposed PodNetworkAttachment, you could achieve this use case. In your implementation it would have to map 1:1 to a Pod, and then you just change Pod for a given PodNetworkAttachment that points to your "sticky IP".